src/core/libs/supl/CMakeLists.txt

```
@@ -55,9 +62,10 @@ target_include_directories(core_libs_supl
         ${CMAKE_CURRENT_SOURCE_DIR}
         ${CMAKE_SOURCE_DIR}/src/core/libs/supl/asn-supl
         ${CMAKE_SOURCE_DIR}/src/core/libs/supl/asn-rrlp
+        ${CMAKE_SOURCE_DIR}/src/core/libs/supl/types
```

This change is consistent with the surrounding lines, however there is an opportunity to improve all of them (I usually fix too much, so feel free to ignore this). A more maintainable change would be 
```suggestion
        ${CMAKE_CURRENT_SOURCE_DIR}/types
```

---

That's nice, I will apply that before merging.